content/src/roadmaps/industries/generic.json

```diff
@@ -35,9 +35,9 @@
       "task": "get-insurance"
     }
   ],
-  "name": "Any Other Business Type",
+  "name": "All Other Businesses",
```

Comment:
Is there a reason we are renaming this industry? I don't see that called out in the tickets.